internal/http/services/owncloud/ocs/data/capabilities.go

```diff
@@ -80,7 +80,8 @@ type CapabilitiesCore struct {
 
 // CapabilitiesGraph holds the graph capabilities
 type CapabilitiesGraph struct {
-	PersonalDataExport ocsBool `json:"personal-data-export" xml:"personal-data-export" mapstructure:"personal_data_export"`
+	PersonalDataExport ocsBool  `json:"personal-data-export" xml:"personal-data-export" mapstructure:"personal_data_export"`
+	ReadOnlyAttributes []string `json:"read_only_attributes" xml:"read_only_attributes" mapstructure:"read_only_attributes"`
```

Comment:
Can we rename this to ReadOnlyUserAttributes? We might need defined separate settings for groups (or other graph resources) at some point.

Reply:
Ah, sorry, I missed the "User" part. *facepalm*

Updated.